File: Ejercicios/C++/Ejercicio6.cpp
#include <iostream>
#include <vector>
using namespace std;
class Prod { public: string nom; int cant; double pre;
    Prod(string n, int c, double p) : nom(n), cant(c), pre(p) {}
};
int main() {
    vector<Prod> inv; 
    inv.push_back(Prod("Mouse", 10, 15.5));
    double total = 0;
    for(Prod p : inv) total += (p.cant * p.pre);
    cout << "Valor inventario: $" << total << "\n";
    return 0;
}

File: Ejercicios/Java/Ejercicio6.java
import java.util.ArrayList;
class Prod {
    String nom; int cant; double pre;
    Prod(String n, int c, double p) { nom=n; cant=c; pre=p; }
}
public class Ejercicio6 {
    public static void main(String[] args) {
        ArrayList<Prod> inv = new ArrayList<>();
        inv.add(new Prod("Mouse", 10, 15.5));
        double total = 0;
        for(Prod p : inv) total += (p.cant * p.pre);
        System.out.println("Valor inventario: $" + total);
    }
}
